2-collectors/scala-stream-collector/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorService.scala

```diff
@@ -157,6 +160,18 @@ class CollectorService(
       case None => HttpResponse(404, entity = "404 not found")
     }
 
+
+  override def rootResponse: HttpResponse =
+    rootResponse(config.rootResponse)
+
+  def rootResponse(c: RootResponseConfig): HttpResponse =
+    if (c.enabled) {
+      val rawHeaders = if (c.headers.nonEmpty) c.headers.map { case (k, v) => RawHeader(k, v) }.toSeq.to[immutable.Seq] else Nil
```

Comment:
why is `to[immutable.Seq]` needed?

Reply:
no need for the `if (c.headers.nonEmpty)` btw

Reply:
@BenFradet I accidentally kept the `toSeq` in there, but the `to[immutable.Seq]` is necessary unless there is an implicit that I should add instead? Without the `to[immutable.Seq]` I get the following compilation error:

```
[error] /Users/rbolkey/src/onespot/snowplow/2-collectors/scala-stream-collector/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/CollectorService.scala:170: type mismatch;
[error]  found   : Seq[akka.http.scaladsl.model.headers.RawHeader]
[error]  required: scala.collection.immutable.Seq[akka.http.scaladsl.model.HttpHeader]
[error]       HttpResponse(c.statusCode, rawHeaders, HttpEntity(c.body))
```

Reply:
I can drop the if check. The default for the headers parameter to `HttpResponse` is `Nil`, so I wasn't sure if it was wasteful to pass an empty map.

Reply:
mmh weird, I'll have a look

Reply:
Fixed it with:

```scala
val rawHeaders = c.headers.map { case (k, v) => RawHeader(k, v) }.toList
```

`Seq` are truly evil :(